packit_service/config.py

```diff
@@ -302,14 +306,18 @@ def get_package_config_from_repo(
                 spec_file_path=spec_file_path,
             )
             if not package_config and fail_when_missing:
-                raise PackitConfigException(
-                    f"No config file found in {project_to_search_in.full_repo_name} "
-                    f"on ref '{reference}'"
+                raise PackitMissingConfigException(
+                    f"No config file for packit (e.g. `.packit.yaml`) found in "
+                    f"{project_to_search_in.full_repo_name} on commit '{reference}'"
                 )
         except PackitConfigException as ex:
+            header = (
+                f"{str(ex)}\n\n"
+                if isinstance(ex, PackitMissingConfigException)
+                else f"Failed to load packit config file:\n```\n{str(ex)}\n```\n"
+            )
```

Comment:
What about `message` instead of a `header` since that's not entirely correct?

```suggestion
            message = (
                f"{str(ex)}\n\n"
                if isinstance(ex, PackitMissingConfigException)
                else f"Failed to load packit config file:\n```\n{str(ex)}\n```\n"
            )
```